composer.json

```diff
@@ -22,37 +22,36 @@
         }
     },
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
         "ext-mbstring": "*",
         "imagine/imagine": "^1.3.2",
-        "symfony/filesystem": "^5.3|^6.0|^7.0",
-        "symfony/finder": "^5.3|^6.0|^7.0",
-        "symfony/framework-bundle": "^5.3|^6.0|^7.0",
-        "symfony/mime": "^5.3|^6.0|^7.0",
-        "symfony/options-resolver": "^5.3|^6.0|^7.0",
-        "symfony/process": "^5.3|^6.0|^7.0",
+        "symfony/filesystem": "^5.4|^6.4|^7.0",
+        "symfony/finder": "^5.4|^6.4|^7.0",
+        "symfony/framework-bundle": "^5.4|^6.4|^7.0",
+        "symfony/mime": "^5.4|^6.4|^7.0",
+        "symfony/options-resolver": "^5.4|^6.4|^7.0",
+        "symfony/process": "^5.4|^6.4|^7.0",
         "twig/twig": "^2.9|^3.0"
     },
     "require-dev": {
         "ext-gd": "*",
         "amazonwebservices/aws-sdk-for-php": "^1.0",
-        "aws/aws-sdk-php": "^2.4|^3.0",
-        "doctrine/persistence": "^1.3|^2.0",
-        "league/flysystem": "^1.0|^2.0|^3.0",
+        "aws/aws-sdk-php": "^3.0",
+        "doctrine/persistence": "^2.0",
+        "league/flysystem": "^3.0",
         "phpstan/phpstan": "^1.10",
         "phpstan/phpstan-symfony": "^1.0",
-        "psr/cache": "^1.0|^2.0|^3.0",
-        "psr/log": "^1.0",
-        "symfony/browser-kit": "^5.3|^6.0|^7.0",
-        "symfony/cache": "^5.3|^6.0|^7.0",
-        "symfony/console": "^5.3|^6.0|^7.0",
-        "symfony/dependency-injection": "^5.3|^6.0|^7.0",
-        "symfony/form": "^5.3|^6.0|^7.0",
-        "symfony/messenger": "^5.3|^6.0|^7.0",
-        "symfony/phpunit-bridge": "^5.3|^6.0|^7.0",
-        "symfony/templating": "^5.3|^6.0|^7.0",
-        "symfony/validator": "^5.3|^6.0|^7.0",
-        "symfony/yaml": "^5.3|^6.0|^7.0"
+        "psr/cache": "^3.0",
```

Comment:
this breaks the symfony 5.4 install. but as this is a new version, i think we should just go with symfony 6.4|7.0 only. the 2.x branch is fine and did not see tons of changes, so people stuck on symfony 5 can keep using that.